src/ports/postgres/modules/utilities/path.sql_in

```diff
@@ -531,7 +548,7 @@ CREATE OR REPLACE FUNCTION MADLIB_SCHEMA.path(
     symbol_expr         VARCHAR,
     pattern_expr        VARCHAR
 ) RETURNS TEXT AS $$
-    SELECT MADLIB_SCHEMA.path($1, $2, $3, $4, $5, $6, NULL, NULL)
+    SELECT MADLIB_SCHEMA.path($1, $2, $3, $4, $5, $6, NULL, NULL, NULL)
 $$ LANGUAGE SQL
 m4_ifdef(`__HAS_FUNCTION_PROPERTIES__', `MODIFIES SQL DATA', `');
 
```

Comment:
Having an example with overlapping_patterns = TRUE might be useful in the
user doc. The same example showing the output with and without overlapping
patterns might be helpful.